addons/website/static/tests/tours/page_manager.js

```diff
@@ -0,0 +1,62 @@
+/** @odoo-module **/
+
+import wTourUtils from 'website.tour_utils';
+
+const delete_selected_page = [
+    {
+        content: "Pick Action",
+        trigger: '.o_cp_action_menus button',
+        run: 'click',
+    },
+    {
+        content: "Pick Delete",
+        trigger: '.o-dropdown--menu span:contains("Delete")',
+        run: 'click',
+    },
+    {
+        content: "Pick I am sure about this",
+        trigger: 'main.modal-body input[type="checkbox"]',
+        run: 'click',
+    },
+    {
+        content: "Pick OK",
+        trigger: '.modal-content footer button.btn-primary',
+        run: 'click',
+}];
+wTourUtils.registerWebsitePreviewTour('website_page_manager', {
+    test: true,
+    url: '/',
+}, [
+    {
+        content: "Pick Site",
+        trigger: 'button.dropdown-toggle[title="Site"]',
+        run: 'click',
+    },
+    {
+        content: "Pick Pages",
+        trigger: 'a.dropdown-item[data-menu-xmlid="website.menu_website_pages_list"]',
+        run: 'click',
+    },
+    {
+        content: "Pick Home Page",
+        trigger: '.o_list_renderer tr:contains("Home"):has(td[name="website_id"]:empty) td.o_list_record_selector input[type="checkbox"]',
+        run: 'click',
+    },
+    ...delete_selected_page,
+    {
+        content: "Check that the page has been removed",
+        trigger: '.o_list_renderer:not(:has(tr:contains("Home"):has(td[name="website_id"]:empty)))',
```

Comment:
You could extract `tr:contains("Home"):has(td[name="website_id"]:empty)` to a variable to avoid duplicating it, it will help if this need to be changed/fixed later (generally avoid duplicate especially when long/complicated stuff like here the selector)